src/Microsoft.Data.SqlClient/ref/Microsoft.Data.SqlClient.cs

```diff
@@ -0,0 +1,2569 @@
+        /// <include file='../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientFactory.xml' path='docs/members[@name="SqlClientFactory"]/CreateConnectionStringBuilder/*'/>
+        public override System.Data.Common.DbConnectionStringBuilder CreateConnectionStringBuilder() { throw null; }
+        /// <include file='../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientFactory.xml' path='docs/members[@name="SqlClientFactory"]/CreateDataAdapter/*'/>
+        public override System.Data.Common.DbDataAdapter CreateDataAdapter() { throw null; }
+#if NETFRAMEWORK
+        /// <include file='../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientFactory.xml' path='docs/members[@name="SqlClientFactory"]/CreateDataSourceEnumerator/*'/>
+        public override System.Data.Common.DbDataSourceEnumerator CreateDataSourceEnumerator() { throw null; }
+
+#endif // NETFRAMEWORK
+        /// <include file='../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientFactory.xml' path='docs/members[@name="SqlClientFactory"]/CreateParameter/*'/>
+        public override System.Data.Common.DbParameter CreateParameter() { throw null; }
+#if NETFRAMEWORK
+        /// <include file='../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientFactory.xml' path='docs/members[@name="SqlClientFactory"]/CreatePermission/*'/>
+        public override System.Security.CodeAccessPermission CreatePermission(System.Security.Permissions.PermissionState state) { throw null; }
+        object System.IServiceProvider.GetService(System.Type serviceType) { throw null; }
+#endif // NETFRAMEWORK
+    }
+#if NETFRAMEWORK
+    /// <include file='../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientLogger.xml' path='docs/members[@name="SqlClientLogger"]/SqlClientLogger/*'/>
+    public partial class SqlClientLogger
+    {
+        /// <include file='../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientLogger.xml' path='docs/members[@name="SqlClientLogger"]/ctor/*'/>
+        public SqlClientLogger() { }
+        /// <include file='../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientLogger.xml' path='docs/members[@name="SqlClientLogger"]/IsLoggingEnabled/*'/>
+        public bool IsLoggingEnabled { get { throw null; } }
+        /// <include file='../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientLogger.xml' path='docs/members[@name="SqlClientLogger"]/LogAssert/*'/>
+        public bool LogAssert(bool value, string type, string method, string message) { throw null; }
+        /// <include file='../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientLogger.xml' path='docs/members[@name="SqlClientLogger"]/LogError/*'/>
+        public void LogError(string type, string method, string message) { }
+        /// <include file='../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientLogger.xml' path='docs/members[@name="SqlClientLogger"]/LogInfo/*'/>
+        public void LogInfo(string type, string method, string message) { }
+    }
+#endif //NETFRAMEWORK
```

Comment:
SqlClientLogger exists in both projects.

Reply:
I think the code exists in NetCore too, it was not exposed for NetCore/NetStandard 🤔 

@JRahnama @DavoudEshtehari it probably got missed with EventSource changes, any concerns exposing this for NetCore/NetStandard refs?

Reply:
I made all the changes except for this one. Once I get a confirmation that I can expose this logger, I'll make another commit to remove the ifdef.

Reply:
I was going to open another PR to merge the `SqlClientLogger.cs`, but it's already in shared src and both netcore and netfx are using it and I see references in netcore, so I think I won't need to open another PR and can make the changes here in the next commit to remove the `ifdef`